test_rmw_implementation/test/test_publisher.cpp

```diff
@@ -42,8 +42,8 @@ class CLASSNAME (TestPublisher, RMW_IMPLEMENTATION) : public ::testing::Test
     context = rmw_get_zero_initialized_context();
     ret = rmw_init(&init_options, &context);
     ASSERT_EQ(RMW_RET_OK, ret) << rcutils_get_error_string().str;
-    const char * const node_name = "my_test_node";
-    const char * const node_namespace = "/my_test_ns";
+    constexpr const char * node_name = "my_test_node";
```

Comment:
```suggestion
    constexpr char node_name[] = "my_test_node";
```

Reply:
Sure, yeah. But for the record, that may force a stack allocation we don't need.

Reply:
For posterity, it turns out that, in exchange, using `constexpr` precludes string literals entirely and reduces somewhat the size of the binary. Thanks for the tip @brawner !